src/wp-admin/update-core.php

```diff
@@ -748,7 +748,7 @@ function list_theme_updates() {
 				<input type="checkbox" name="checked[]" id="<?php echo $checkbox_id; ?>" value="<?php echo esc_attr( $stylesheet ); ?>" />
 				<label for="<?php echo $checkbox_id; ?>" class="screen-reader-text">
 					<?php
-					/* translators: %s: Theme name. */
+					/* translators: %hidden accessibility text. %s: Theme name. */
```

Comment:
```suggestion
					/* translators: Hidden accessibility text. %s: Theme name. */
```
Looks like a small mistake here 🙂